packages/firebase_crashlytics/firebase_crashlytics/test/mock.dart

```diff
@@ -2,18 +2,16 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.9
-
 import 'package:firebase_core_platform_interface/firebase_core_platform_interface.dart';
 import 'package:firebase_crashlytics_platform_interface/firebase_crashlytics_platform_interface.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-typedef Callback(MethodCall call);
+typedef Callback = void Function(MethodCall call);
 
 final List<MethodCall> methodCallLog = <MethodCall>[];
 
-setupFirebaseCrashlyticsMocks([Callback /*?*/ customHandlers]) {
+dynamic setupFirebaseCrashlyticsMocks([Callback? customHandlers]) {
```

Comment:
Nothing is actually returned from this function, so it can just be `void` instead of `dynamic`